modules/bbp-core/src/main/resources/schemas/bbp/core/brainLocation/v0.1.0.json

```diff
@@ -0,0 +1,117 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbpvect3dsh": "{{base}}/schemas/bbp/core/vector3d/v0.1.0/shapes/",
+      "bbptlotsh": "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/core/brainlocation/v0.1.0/shapes/"
+    }
+  ],
+  "imports": [
+    "{{base}}/schemas/bbp/core/vector3d/v0.1.0"
+  ],
+  "@type": "owl:Ontology",
+  "shapes": [
+    {
+      "@id": "this:BrainLocationShape",
+      "@type": "sh:NodeShape",
+      "label": "Brain location shape definition",
+      "property": [
+        {
+          "path": "nsg:brainRegion",
+          "name": "Brain region",
+          "description": "Location as brain region",
+          "node": "this:BrainRegionShape",
```

Comment:
Why not adding directly the following shape here: "node": "bbptlotsh:BrainRegionOntologyTermShape" ?

Reply:
I thought it might give us more freedom if we ever wanted to change the BrainRegionShape (e.g. to include a property shape that allows to explicitely state the layer).

Reply:
Any change we will make with regard to how to represent brain region, should be consistent across all of data. Which means the brain region representation should always be centralised in the [typedlabeledontologyterm schema] (https://github.com/BlueBrain/nexus-bbp-domains/blob/master/modules/bbp-core/src/main/resources/schemas/bbp/core/typedlabeledontologyterm/v0.1.0.json)